core/backend/src/test/changesets/ChangeMerging.test.ts

```diff
@@ -1,142 +0,0 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-
-import { assert } from "chai";
-import * as path from "path";
-import { Id64String, OpenMode } from "@itwin/core-bentley";
```

Comment:
@pmconne after consulting with @swwilson-bsi  we found ChangeMerging is a very old test used to test no-lock merge which was something on the table very early in itwin.js development. My new test and this test do the same thing both use no-lock to change elements and expect them to either merge or fail. 

This test failed with the expected error but we decided to remove it as its redundant to what I have.